### ISSUE TITLE
Signal has a "fizz"

### ISSUE THREAD

**Issue by cyrusadkisson** [brummer10/XDarkTerreor.lv2#4]
Unfortunately, there is a deal-breaking "shhhh" fizz that you get regardless of the gain level when using this plugin.

**Comment by brummer10** [comment]
Do you using a arm device?

**Comment by cyrusadkisson** [comment]
Yes. I wasn't aware that cpu architecture could matter for these types of things.

**Comment by cyrusadkisson** [comment]
And BTW, I get the same fizz on my desktop x86_64 machine. Not sure it does matter.

**Comment by brummer10** [comment]
you should use some optimization flags for your ARM FPU register. 
like 
`-march=armv7-a -mfpu=vfpv3`

> And BTW, I get the same fizz on my desktop x86_64 machine. Not sure it does matter.

maybe your sound-card forward dc-offset , (a lowpass filter in the sound-card at 21Hz helps a lot in such cases), that may be lead to some fizzing sound with this plug, I could otherwise implement a dcblocker to see if that helps.

**Comment by cyrusadkisson** [comment]
I don't have this issue with any of the other amp plugins (supersonic, epic, gxamp) on the same hardware (both arm and x64).  It's not a noise being picked up by the pickups or anything like that. There is something wrong in the signal processing inside. BTW I only get the fizz when playing. 

The non-dark orange terror plugin also does this, although it's not as pronounced. (Edit: resolved. It's tiny terror's fizz was insignificant and matches my Line 6 Helix.)

Have you tried it? Do you hear it?

**Comment by brummer10** [comment]
It's not a noise picked up I talk about. It's dc-offset, this plugs suffers from that. It's related to the dsp algorithm they use.
Let me implement the dc-blocker and we''ll see. 
I'll send a note here when done.

**Comment by brummer10** [comment]
On thing I forgot to mention, those XTerror plugs **must** be used with a cabinet convolver, as they are plain head simulations. 
Otherwise they wont sound any good.
https://github.com/brummer10/XDarkTerror.lv2#features

**Comment by cyrusadkisson** [comment]
Yes I'm using a cabinet convolver.

**Comment by cyrusadkisson** [comment]
Any thoughts about how to get rid of the fizz I was hearing?
